test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,109 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+const md5 = require('md5');
+
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+
+chai.use(chaiSubset);
+
+const path = 'https://api.github.com/users/aperdomob';
+
+const user = {
+  name: 'Alejandro Perdomo',
+  company: 'Perficient Latam',
+  location: 'Colombia'
+};
+
+const jasmineRepository = {
+  full_name: 'aperdomob/jasmine-json-report',
+  private: false,
+  description: 'A Simple Jasmine JSON Report'
+};
+
+const readme = {
+  name: 'README.md',
+  path: 'README.md',
+  sha: '360eee6c223cee31e2a59632a2bb9e710a52cdc0'
+};
+
+let jasmineReportRepository;
+let md5downloaded;
+let readmeData;
+let md5downloadedReadme;
+
+let serviceHypermedia;
+let repositories;
+let downloadedRepo;
+let repositoryContent;
+let downloadedReadme;
+
+describe('GET methods', () => {
+  before(async () => {
+    serviceHypermedia = await axios.get(`${path}`);
+  });
+
+  it('User verification', async () => {
```

Comment:
async not need it because you're not `await`ing anything inside `it`

Reply:
resolved